src/org/zaproxy/zap/extension/api/API.java

```diff
@@ -315,7 +314,7 @@ public boolean handleApiRequest (HttpRequestHeader requestHeader, HttpInputStrea
 								}
 							}
 						}
-						
+
 						res = impl.handleApiOptionAction(name, params);	
```

Comment:
This indentation change should be reverted. [1]

[1] https://github.com/zaproxy/zaproxy/wiki/DevGuidelines